Selenium.psm1

```diff
@@ -106,14 +106,22 @@ function Start-SeChrome {
             $Driver = New-Object -TypeName "OpenQA.Selenium.Chrome.ChromeDriver" -ArgumentList $AssembliesPath,$Chrome_Options
         }
         else{
-            $Driver = New-Object -TypeName "OpenQA.Selenium.Chrome.ChromeDriver" -ArgumentList $Chrome_Options 
+            $Driver = New-Object -TypeName "OpenQA.Selenium.Chrome.ChromeDriver" -ArgumentList $Chrome_Options
         }
 
         if($Minimized){
             $driver.Manage().Window.Minimize();
 
         }
 
+        if($Headless -and $DefaultDownloadPath) {
+            $CmdParams = New-Object 'system.collections.generic.dictionary[[System.String],[System.Object]]]'
+            $CmdParams.Add('behavior', 'allow')
+            $CmdParams.Add('downloadPath', $DefaultDownloadPath.FullName)
+
```

Comment:
@Windos @adamdriscoll I'm thinking maybe changing the name of the Params variable to something like $HeadlessDownloadParams or something like that will be better since it will make it recognizable with the headless download feature.
what do you guys think?

Reply:
Me too, I reverted those unrelated formatting changes and changed the variable name.